src/main/java/bio/terra/service/snapshot/SnapshotDao.java

```diff
@@ -599,10 +604,12 @@ public SnapshotSummary retrieveSummaryById(UUID id) {
   public List<SnapshotSummary> retrieveSnapshotsForDataset(UUID datasetId) {
     try {
       String sql =
-          "SELECT snapshot.id, name, description, created_date, profile_id, "
+          "SELECT s.id, s.name, s.description, s.created_date, s.profile_id, "
+              + "d.security_classification, "
               + snapshotSourceStorageQuery
-              + "FROM snapshot "
-              + "JOIN snapshot_source ON snapshot.id = snapshot_source.snapshot_id "
+              + "FROM snapshot s "
+              + "JOIN snapshot_source ON s.id = snapshot_source.snapshot_id "
+              + "JOIN dataset d ON d.id = snapshot_source.dataset_id "
```

Comment:
What is the driving factor for using the "s" and "d" aliases?

Reply:
Since we're now querying across datasets and snapshots, we need to disambiguate columns.

Reply:
We could just call snapshot.<field> and dataset.<field>, though, right?

Reply:
You know, good point. I'll do that.